### ISSUE TITLE
Segment error (core dumped)      段错误 (核心已转储)

### ISSUE THREAD

**Issue by Lirene3329** [zju3dv/PGSR#59]
(shuziren) camera@camera:~/桌面/pgsr-main$ python render.py -m /home/camera/Camera_test/undistort/output --max_depth 5.0 --voxel_size 0.008 --mask_path /home/camera/Camera_test/undistort/mask
Looking for config file in /home/camera/Camera_test/undistort/output/cfg_args
Config file found: /home/camera/Camera_test/undistort/output/cfg_args
Rendering /home/camera/Camera_test/undistort/output
multi_view_num 8 [30/10 13:46:14]
Loading trained model at iteration 10000 [30/10 13:46:14]
Reading camera 32/32 [30/10 13:46:14]
Converting point3d.bin to .ply, will happen only the first time you open the scene. [30/10 13:46:14]
xyz (17841, 3) [30/10 13:46:14]
cameras_extent 4.774346828460693 [30/10 13:46:14]
Loading Training Cameras [30/10 13:46:14]
[ INFO ] Encountered quite large input images (>1.6K pixels width), rescaling to 1.6K.
 If this is not desired, please explicitly specify '--resolution/-r' as 1 [30/10 13:46:14]
scale 2.564375 [30/10 13:46:14]
load camera 31Loading Test Cameras [30/10 13:46:18]
computing nearest_id [30/10 13:46:18]
TSDF voxel_size 0.008 [30/10 13:46:18]
Rendering progress:   0%|                                                                                       | 0/32 [00:00<?, ?it/s]/home/camera/anaconda3/envs/shuziren/lib/python3.10/site-packages/torch/functional.py:534: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /opt/conda/conda-bld/pytorch_1729647352509/work/aten/src/ATen/native/TensorShape.cpp:3595.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Rendering progress: 100%|██████████████████████████████████████████████████████████████████████████████| 32/32 [00:02<00:00, 14.55it/s]
TSDF Fusion progress:   0%|                                                                                     | 0/32 [00:00<?, ?it/s]段错误 (核心已转储)

**I encountered a section error problem when running, please tell me how I should solve this problem? thank you**

**Comment by samutou** [comment]
Hello, I am also experiencing the same problem, is it possible to No tell me your solution?
